hyperspy/component.py

```diff
@@ -1213,8 +1213,8 @@ def as_dictionary(self, fullcopy=True):
         export_to_dictionary(self, self._whitelist, dic, fullcopy)
         from hyperspy.model import _COMPONENTS
         if self._id_name not in _COMPONENTS:
-            import dill
-            dic['_class_dump'] = dill.dumps(self.__class__)
+            import cloudpickle
```

Comment:
Can `cloudpickle` deserialize object serialized with `dill` and viceversa? Notice that the serialized objects can be saved/loaded to/from files.

Reply:
I assume (wishful thinking?) that this would be covered by the test suite but didn't check! 😅